cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -1214,21 +1221,29 @@ data ChainInformation = ChainInformation
   , blocksUncoupledDelta :: Int64
     -- ^ The net change in number of blocks forged since last restart not on the
     -- current chain.
+  , fork :: Bool
+    -- ^ Was this a fork.
   }
 
 chainInformation
   :: forall blk. HasHeader (Header blk)
   => ChainDB.NewTipInfo blk
+  -> Maybe (AF.AnchoredFragment (Header blk))
   -> AF.AnchoredFragment (Header blk)
   -> Int64
   -> ChainInformation
-chainInformation newTipInfo frag blocksUncoupledDelta = ChainInformation
-    { slots = unSlotNo $ fromWithOrigin 0 (AF.headSlot frag)
-    , blocks = unBlockNo $ fromWithOrigin (BlockNo 1) (AF.headBlockNo frag)
-    , density = fragmentChainDensity frag
+chainInformation newTipInfo oldFrag_m newFrag blocksUncoupledDelta = ChainInformation
```

Comment:
See my related comment in `traceChainMetrics`: we could get by with just a new `Bool` argument instead.